pep-0659.rst

```diff
@@ -349,7 +318,44 @@ None
 Rejected Ideas
 ==============
 
-Too many to list.
+By implementing a specializing adaptive interpreter with inline data caches,
+we are implicitly rejecting many alternative ways to optimize CPython.
+However, it is worth emphasizing that some ideas, such as just-in-time
+compilation, have not been rejected, merely deferred.
+
+Storing data caches before the bytecode.
```

Comment:
Do you think it's worth enumerating a few reasons why we decided not to go this route?

Reply:
Added a brief comment.